tests/Unit/EventSubscriber/GeneratedUserIngestControllerSubscriberTest.php

```diff
@@ -73,12 +73,67 @@ public function testOnLineItemUpdatedIgnoreOnDisabled(): void
             $userFolderSyncMock,
             new CreateUserServiceContext(['test1'], ['tao1'], 10, true),
             'test',
+            false,
+            true
+        );
+
+        $service->onLineItemUpdated(new LineItemUpdated(['test1']));
+    }
+
+    public function testOnLineItemUpdatedIgnoreFolderSyncOnDisabled(): void
+    {
+        $logerMock = $this->createMock(LoggerInterface::class);
+        $createServiceMock = $this->createMock(BulkCreateUsersService::class);
+        $createServiceMock->expects(self::never())->method('generate');
+
+        $generateGroupIdsServiceMock = $this->createMock(GenerateGroupIdsService::class);
+        $ltiInstanceRepositoryMock = $this->createMock(LtiInstanceRepository::class);
+        $lineItemRepositoryMock = $this->createMock(LineItemRepository::class);
+        $userFolderSyncMock = $this->createMock(FolderSyncService::class);
+
+        $service = new GeneratedUserIngestControllerSubscriber(
+            $logerMock,
+            $createServiceMock,
+            $generateGroupIdsServiceMock,
+            $ltiInstanceRepositoryMock,
+            $lineItemRepositoryMock,
+            $userFolderSyncMock,
+            new CreateUserServiceContext(['test1'], ['tao1'], 10, true),
+            'test',
+            false,
             false
         );
 
         $service->onLineItemUpdated(new LineItemUpdated(['test1']));
     }
 
+    public function testOnLineItemUpdatedIgnoreOnEmptyList(): void
```

Comment:
This test shares same source code that `testOnLineItemUpdatedIgnoreFolderSyncOnDisabled ` does. 
I don't understand how do they test different stuff with the same source code.